docs/architecture/adr-012-problem-plugin-architecture.html

```
@@ -0,0 +1,641 @@
+
+  ContentSwapLambda:
+    Type: AWS::Lambda::Function
+    Properties:
+      Code:
+        ZipFile: |
+          import boto3
+          def handler(event, _):
+              variant = event["detail"]["parameters"]["contentVariant"]
+              boto3.client("s3").copy_object(
+                Bucket="<bucket>", Key="index.html",
+                CopySource=f"<bucket>/_staging/{variant}/index.html"
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </bucket></bucket> ], start tag match failed [ <bucket> ] on line 541.

(tag-pair)